index.html

```diff
@@ -5,21 +5,55 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>QUIZ APP</title>
     <link rel="stylesheet" href="./components/index.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600&display=swap"
+      rel="stylesheet"
+    />
     <script
       src="https://kit.fontawesome.com/87c9eac3be.js"
       crossorigin="anonymous"
     ></script>
   </head>
   <body class="app-grid">
-    <section class="header"></section>
+    <section class="header">
+      <h1 class="header__heading p-10">QUIZ APP</h1>
+    </section>
     <section class="content">
       <head></head>
-      <main></main>
+      <main>
+        <!-- Ich komm nicht drauf, warum mein Bookmark sich nicht an der Box als Container orientert, um hier eine absolute Position einzunehmen. Hilfe bitte :-) -->
+
+        <section class="quiz-card quiz-card__box">
```

Comment:
Why do you have two class names here? Add the styles from `quiz-card__box` to your `quiz-card`.
```suggestion
        <section class="quiz-card">
```